ios/sdk/sdk.xcodeproj/project.pbxproj

```diff
@@ -52,7 +53,7 @@
 /* End PBXBuildFile section */
 
 /* Begin PBXFileReference section */
-		03F2ADC957FF109849B7FCA1 /* libPods-JitsiMeet.a */ = {isa = PBXFileReference; explicitFileType = archive.ar; includeInIndex = 0; path = "libPods-JitsiMeet.a"; sourceTree = BUILT_PRODUCTS_DIR; };
+		05646ED4E81A6390797D3086 /* libPods-JitsiMeet.a */ = {isa = PBXFileReference; explicitFileType = archive.ar; includeInIndex = 0; path = "libPods-JitsiMeet.a"; sourceTree = BUILT_PRODUCTS_DIR; };
```

Comment:
@hristoterezov, could you please try manually dropping all the CocoaPods modifications in the Xcode project file like libPods-JitsiMeet, [CP] Copy Pods Resources  etc. I don't see why they are necessary.

Reply:
I would but I'm not really sure what should stay (if any at all). Could you please help me with this? Can I safely remove all the changes here or maybe leave only the Dropbox related ones?

Reply:
All lines mentioning pods, CP, etc. Leave only the ones referencing the new dropbox .h and .m files and their UUIDs. Anyway, I can try that out myself, don't worry.